### DISCUSSION TITLE
Alternative offline file location and shares

### DISCUSSION THREAD

**Discussion by proxypunker** [iterate-ch/cyberduck#16288] [Q&A]
I really like Mountain Duck. I have different cloud space. 
Now under Windows is there a way to set a different file location for offline files?
And is there a way to share the files under windows via smb?
Because I would really love to use it on my private server.

**Comment by dkocher** [comment]
- Support for SMB protocol is forthcoming in version 5. 
- Refer to our [documentation](https://docs.mountainduck.io/mountainduck/preferences/#cache-location) how to set the cache location.

**Reply by proxypunker** [reply]
Thank you very much for your quick response! 
So there's actually no way to say: "I want this specific account on this drive." ?

**Reply by tatze96** [reply]
The cache location can only be set globally for all bookmarks. It isn't possible to specify the cache location on a per bookmark basis.
